### ISSUE TITLE
Add option to put points at the end of the card subject

### ISSUE THREAD

**Issue by mariusursache** [Q42/TrelloScrum#76]
I work with a lot of other people who use Trello on Safari or as FluidApp. The parantheses at the beginning of the subject look odd. Is it possible to have an option to put all the parantheses at the end of the subject? It works/looks ok if done manually.

Example:

Instead of: "(3) This task must be done",
Have this: "This task must be done (3)".

**Comment by SimonEast** [comment]
Yes, I always type the points manually at the end of the title too, to make it more readable for others.

**Comment by twnaing** [comment]
+1

If I had my way, I would prefer

    Task name [5/5]

First number is actual and second number is estimated.
